src/screens/Connectors/PayoutProcessor/PayoutProcessorAccountDetails.res

```diff
@@ -0,0 +1,275 @@
+        | _ => false
+        } &&
+        featureFlagDetails.paypalAutomaticFlow
+      ) {
+        initialValuesToDict->Dict.set(
+          "connector_label",
+          initialValues
+          ->getDictFromJsonObject
+          ->getString("connector_label", "")
+          ->JSON.Encode.string,
+        )
+        initialValuesToDict->Dict.set(
+          "profile_id",
+          initialValuesToDict->getString("profile_id", "")->JSON.Encode.string,
+        )
+      } else if connector->isNonEmptyString {
+        initialValuesToDict->Dict.set(
+          "connector_label",
+          `${connector}_${activeBusinessProfile.profile_name}`->JSON.Encode.string,
+        )
+        initialValuesToDict->Dict.set(
+          "profile_id",
+          activeBusinessProfile.profile_id->JSON.Encode.string,
+        )
+      }
+    }
+    if (
+      connectorTypeFromName->checkIsDummyConnector(featureFlagDetails.testProcessors) &&
+        !isUpdateFlow
+    ) {
+      let apiKeyDict = [("api_key", "test_key"->JSON.Encode.string)]->Dict.fromArray
+      initialValuesToDict->Dict.set("connector_account_details", apiKeyDict->JSON.Encode.object)
+
+      initialValuesToDict->JSON.Encode.object
+    } else {
+      initialValues
+    }
+  }, [connector, activeBusinessProfile.profile_id])
+
+  let onSubmitMain = async values => {
+    open ConnectorTypes
+    try {
+      let body = generateInitialValuesDict(
+        ~values,
+        ~connector,
+        ~bodyType,
+        ~isLiveMode={featureFlagDetails.isLiveMode},
+        ~connectorType=ConnectorTypes.PayoutConnector,
+      )
+      setScreenState(_ => Loading)
+      setCurrentStep(_ => PaymentMethods)
+      setScreenState(_ => Success)
+      setInitialValues(_ => body)
+    } catch {
+    | Exn.Error(e) => {
+        setShowVerifyModal(_ => false)
+        setVerifyDone(_ => ConnectorTypes.NoAttempt)
+        switch Exn.message(e) {
+        | Some(message) => {
+            let errMsg = message->parseIntoMyData
+            if errMsg.code->Option.getOr("")->String.includes("HE_01") {
+              showToast(
+                ~message="This configuration already exists for the connector. Please try with a different country or label under advanced settings.",
+                ~toastType=ToastState.ToastError,
+              )
+              setCurrentStep(_ => IntegFields)
+              setScreenState(_ => Success)
+            } else {
+              showToast(
+                ~message="Failed to Save the Configuration!",
+                ~toastType=ToastState.ToastError,
+              )
+              setScreenState(_ => Error(message))
+            }
+          }
+
+        | None => setScreenState(_ => Error("Failed to Fetch!"))
+        }
+      }
+    }
+  }
+
+  let onSubmitVerify = async values => {
+    try {
+      let body =
+        generateInitialValuesDict(
+          ~values,
+          ~connector,
+          ~bodyType,
+          ~isLiveMode={featureFlagDetails.isLiveMode},
```

Comment:
connector type should be passed here as prop for the function 
`~connectorType=ConnectorTypes.PayoutConnector`